src/Files.App/Styles/ThemedIconSet.xaml

```diff
@@ -0,0 +1,4568 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Files.App.UserControls">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="ms-appx:///UserControls/ThemedIcon/ThemedIcon.xaml" />
+		<ResourceDictionary>
+
+			<!--  Empty  -->
+			<Style
+				x:Name="TIcon_Empty"
+				BasedOn="{StaticResource DefaultThemedIconStyle}"
+				TargetType="local:ThemedIcon">
+
+				<Setter Property="Width" Value="16" />
+				<Setter Property="Height" Value="16" />
+
+				<Setter Property="OutlineIconData" Value="" />
+				<Setter Property="FilledIconData" Value="" />
+
+				<Setter Property="Layers">
+					<Setter.Value>
+						<local:ThemedIconLayers>
+							<!--  Place lower layers first in the list  -->
+							<local:ThemedIconLayer LayerType="Alt" PathData="" />
+							<local:ThemedIconLayer LayerType="Base" PathData="" />
+							<local:ThemedIconLayer LayerType="Accent" PathData="" />
+							<local:ThemedIconLayer LayerType="AccentContrast" PathData="" />
+						</local:ThemedIconLayers>
+					</Setter.Value>
+				</Setter>
+			</Style>
+
+			<!--#region  Common Icons-->
```

Comment:
In general we stay away from regions but I agree that it might be useful here.

Reply:
ThemedIcons.Common.xaml makes you guys hard to follow?

Reply:
You would end up with several Xaml files containing icon styles, which may or may not add confusion with autocomplete dot separated style names.

Reply:
I dont think so?

Reply:
I have to agree with Martin here

Reply:
The naming of the icons, as well as the regions for logical separation, is enough to keep on top of the amount of xaml there is, and also adding new icons, only means finding the right region to add to it.  And if the icon xaml is put in the "wrong" xaml region, its in the right file, and the naming will still work as expected.